src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseClient.java

```diff
@@ -124,7 +101,7 @@ CollectionExerciseDTO getCollectionExercise(final UUID collexId) throws CTPExcep
           .getBody();
     } catch (UnirestException e) {
       throw new CTPException(
-          CTPException.Fault.SYSTEM_ERROR, "Failed to get collection exercise", e);
+          CTPException.Fault.SYSTEM_ERROR, "Failed to get collection exercise: %s", e);
```

Comment:
This is so that the UnirestException is captured in the error message. Without the %s we're not capturing the actual exception being thrown.

Reply:
I don't think it give us anymore value because UnirestException doesn't implement toString

Reply:
@benjefferies In all fairness, I added the %s to diagnose an integration test failure in collection exercise. Because it's intermittent I need more information on how to stabilise it.